data/json/npcs/talk_tags.json

```
@@ -523,13 +604,28 @@
       "Put down the <yrwp>!",
       "Drop your <swear> weapon, <name_b>!",
       "Put down your <yrwp>!",
-      "Alright, drop the <yrwp>!"
+      "Alright, drop the <yrwp>!",
+      "Please put down your weapon. You have 20 seconds to comply.",
```

Again with the 20 seconds thing. That makes it sound like you have 20 turns until they shoot you, which you don't. Also, redundancy (not that anyone would notice).
```suggestion
      "Please put down your weapon. I'll give you to the count of three. One...",
```
Also, this doesn't match the tone of the previous snippets -- keep in mind that these can show up when you're being mugged.